app/src/main/java/org/eyeseetea/malariacare/data/MediaModeLocalDataSource.java

```diff
@@ -6,15 +6,15 @@
 
 import org.eyeseetea.malariacare.R;
 import org.eyeseetea.malariacare.data.database.utils.PreferencesState;
-import org.eyeseetea.malariacare.domain.boundary.IStylePreferencesRepository;
+import org.eyeseetea.malariacare.domain.boundary.IMediaModeRepository;
+import org.eyeseetea.malariacare.domain.entity.Settings;
 
 
-public class StylePreferenceLocalDataSource implements IStylePreferencesRepository {
-    private final static ListStyle DEFAULT_LIST_STYLE = ListStyle.GRID;
+public class MediaModeLocalDataSource implements IMediaModeRepository {
```

Comment:
The repositories should not return primitive values ​but domain entities. If we are going to use Settings Domain Entity, then we should use existed SettingsRepository